os/fs/tmpfs/Kconfig

```diff
@@ -69,5 +70,24 @@ config FS_TMPFS_FILE_FREEGUARD
 		little more memory than needed is always allocated.  This permits
 		the file to shrink without so many realloctions.
 
+config FS_TMPFS_USE_NHEAP
```

Comment:
You don't have to make additional config here.
As possible as we can, we had better reduce subordinative config.
So below is more comfortable option to add new board in future.
```
#ifdef NHEAP_ENABLED 
#define TMPFS_MALLOC(size) malloc_at(2, size);
#else 
#define TMPFS_MALLOC(size) malloc(size);
#endif
```

Reply:
The settings have been moved to Boot Memory Configuration.